components/blog/BlogPost.vue

```diff
@@ -0,0 +1,42 @@
+<template>
+  <b-row
+    class="flex-md-row"
+    data-qa="blog post"
+  >
+    <b-col cols="12">
```

Comment:
Let's wrap the heading and article body in an `<article>` element.

Let's also give it the same styling as the textual content on the static pages, e.g. https://portaljs-test.eanadev.org/about-us, i.e. white background with a border.